pycomlink/validation/stats.py

```diff
@@ -157,12 +155,11 @@ def calc_rain_error_performance_metrics(reference, predicted, rainfall_threshold
     if rainfall_threshold_wet is None:
         rainfall_threshold_wet = 0.1
 
-
     # calculate performance metrics: pcc, cv, rmse and mae
     pearson_correlation = np.corrcoef(reference, predicted)
-    coefficient_of_variation = np.std(predicted)/np.std(reference) - np.mean(reference)
+    coefficient_of_variation = np.std(predicted-reference) - np.mean(reference)
```

Comment:
should be: `coefficient_of_variation = np.std(predicted-reference) / np.mean(reference)`